bin/sngl/pycbc_export_censored_trigs

```diff
@@ -102,8 +105,68 @@ else:
     trigs_censored = ForegroundTriggers(opts.coinc_file, opts.bank_file,
                           sngl_files=opts.sngl_files, group=group)
 
-# find triggers above limit
-mask = numpy.where( trigs_censored.get_coincfile_array("stat") > opts.min_stat )[0]
+
+# Apply clustering to censored_triggers if a cluster-window has been 
+# parsed in the command line.
+
+# Clustering picks the loudest trigger in each cluster-window
+
+if opts.cluster_window:
+    
+    # get newsnr for all censored triggers
+    coinc_newsnr = trigs_censored.get_coincfile_array("stat")
+    
+    # get H1 trigger times from the coinc censored trigger file
+    time1 = trigs_censored.get_coincfile_array("time1")
+    
+    # get the minimum and maximum times for the triggers
+    t_start = numpy.amin(time1)
+    t_end = numpy.amax(time1)
+    
+    # Divide the time range in which the triggers are distributed into 
+    # bins using the cluster-window
+    nbins = (t_end - t_start)/opts.cluster_window
+
+    # set the lower-bound of the 1st time bin to the minimum time for 
+    # the set of triggers
+    t_low = t_start
+
+    cluster_idx = []
+    
+    # Run a loop over the time bins
+    for i in range(int(nbins)):
```

Comment:
This is just an edge case but typecasting ``nbins`` as an ``int`` means you will lose whatever fraction of a second at the end of the analysis time if ``opts.cluster_window`` is not an integer number.

A simple way to get around this is change the argument to something like ``int(nbins)+1`` or ``int(numpy.ceil(nbins))``. Then it should be fine.

Otherwise your changes all look good.